src/components/pages/detailsApplication/ButtonManageGroupSettings/index.tsx

```diff
@@ -65,26 +82,52 @@ export const ButtonManageGroupSettings = ({
 					) + group.name
 				}
 				body={
-					<FormGroup>
-						<FormControlLabel
-							control={
-								<Switch
-									checked={
-										enabledSelfManagedGroup ??
-										false
-									}
-									name="selfmanaged-toggle"
-									color="primary"
-									onChange={
-										handleChangeSelfManagedGroup
+					<Grid container flexDirection="column">
+						<Grid container>
+							<TextField
```

Comment:
Personally, I will wrap each item in <Grid item ws={12}><Grid/>. It could enhance future ui evolution :)